test/browser/sauce-browsers.json

```diff
@@ -1,5 +1,12 @@
 [
   {
+    "browserName": "microsoftedge",
+    "platform": "WIN10",
+    "version": "20.10240"
```

Comment:
I think experimenting with dropping version numbers would be more approp for another pr.